TombEngine/Objects/game_object_ids.h

```diff
@@ -697,6 +697,10 @@ enum GAME_OBJECT_ID : short
 	ID_SEQUENCE_SWITCH3,
 	ID_STEEL_DOOR,
 	ID_GOD_HEAD,
+	ID_BREAKABLE_WALL1,
```

Comment:
Is it necessary to have 4 slots for breakable wall? Isn't it enough to have one slot, and meshswap a given object if you need variation?